frontend/src/components/Agreements/AgreementBLIAccordion/AgreementBLIAccordion.jsx

```diff
@@ -45,9 +51,16 @@ function AgreementBLIAccordion({
     setAfterApproval,
     action,
     isApprovePage = false,
-    updatedBudgetLines = []
+    updatedBudgetLines = [],
+    changeRequestType = undefined,
+    newAwardingEntity = undefined,
+    oldAwardingEntity = undefined
 }) {
-    const showToggle = action === selectedAction.DRAFT_TO_PLANNED || action === BLI_STATUS.PLANNED || isApprovePage;
+    const showToggle =
+        action === selectedAction.DRAFT_TO_PLANNED ||
+        action === BLI_STATUS.PLANNED ||
+        changeRequestType === CHANGE_REQUEST_SLUG_TYPES.PROCUREMENT_SHOP ||
+        isApprovePage;
```

Comment:
think the isApprove page would cover our scenario and changeRequestType is not needed